### ISSUE TITLE
Segfault with portable linux executable on Ubuntu 16.04

### ISSUE THREAD

**Issue by Timmmm** [pawelsalawa/sqlitestudio#3423]
Running Ubuntu 16.04, I downloaded the portable linux version of SQLiteStudio 3.2.1, ran `./sqlitestudio` and it immediately gives a segfault.

I debugged it and the backtrace is as follows. Since it is some X-related thing I suspect this is because I am running it under `Xvnc`. Maybe `Xvnc` doesn't support some X11 extension that Qt expects and it just segfaults...

```
Thread 1 "sqlitestudio" received signal SIGSEGV, Segmentation fault.
0x00007fffee221ef1 in QXcbConnection::internAtom(char const*) ()
   from /home/timh/local/SQLiteStudio/platforms/../lib/libQt5XcbQpa.so.5
(gdb) bt
#0  0x00007fffee221ef1 in QXcbConnection::internAtom(char const*) ()
   from /home/timh/local/SQLiteStudio/platforms/../lib/libQt5XcbQpa.so.5
#1  0x00007fffee23598b in QXcbScreen::getEdid() const ()
   from /home/timh/local/SQLiteStudio/platforms/../lib/libQt5XcbQpa.so.5
#2  0x00007fffee236f72 in QXcbScreen::QXcbScreen(QXcbConnection*, QXcbVirtualDesktop*, unsigned int, xcb_randr_get_output_info_reply_t*, xcb_xinerama_screen_info_t const*, int) ()
   from /home/timh/local/SQLiteStudio/platforms/../lib/libQt5XcbQpa.so.5
#3  0x00007fffee223f41 in QXcbConnection::initializeScreens() ()
   from /home/timh/local/SQLiteStudio/platforms/../lib/libQt5XcbQpa.so.5
#4  0x00007fffee224fbf in QXcbConnection::QXcbConnection(QXcbNativeInterface*, bool, unsigned int, char const*) ()
   from /home/timh/local/SQLiteStudio/platforms/../lib/libQt5XcbQpa.so.5
#5  0x00007fffee22add4 in QXcbIntegration::QXcbIntegration(QStringList const&, int&, char**) ()
   from /home/timh/local/SQLiteStudio/platforms/../lib/libQt5XcbQpa.so.5
#6  0x00007fffee59243d in ?? ()
   from /home/timh/local/SQLiteStudio/platforms/libqxcb.so
#7  0x00007ffff41f489b in QPlatformIntegrationFactory::create(QString const&, QStringList const&, int&, char**, QString const&) ()
   from /home/timh/local/SQLiteStudio/lib/libQt5Gui.so.5
#8  0x00007ffff4202c51 in QGuiApplicationPrivate::createPlatformIntegration()
    () from /home/timh/local/SQLiteStudio/lib/libQt5Gui.so.5
#9  0x00007ffff420370d in QGuiApplicationPrivate::createEventDispatcher() ()
   from /home/timh/local/SQLiteStudio/lib/libQt5Gui.so.5
#10 0x00007ffff6162ba1 in QCoreApplicationPrivate::init() ()
   from /home/timh/local/SQLiteStudio/lib/libQt5Core.so.5
#11 0x00007ffff42051ab in QGuiApplicationPrivate::init() ()
   from /home/timh/local/SQLiteStudio/lib/libQt5Gui.so.5
#12 0x00007ffff6b1f369 in QApplicationPrivate::init() ()
   from /home/timh/local/SQLiteStudio/lib/libQt5Widgets.so.5
#13 0x000000000040bba4 in SingleApplication::SingleApplication(int&, char**, bool, QFlags<SingleApplication::Mode>, int) ()
#14 0x0000000000405a23 in main ()
```

**Comment by Timmmm** [comment]
Ah looks like this was [a Qt bug](https://bugreports.qt.io/browse/QTBUG-64928) that has been fixed in version 5.10, so I guess your version of Qt just needs updating.

**Comment by pawelsalawa** [comment]
Fixed in 5.10.1 exactly. 5.10.0 (shipped with SQLiteStudio) is still affected. Bug confirmed.

**Comment by pawelsalawa** [comment]
Portable distribution of SQLiteStudio 3.2.2 got upgraded to Qt 5.14.2.
